aws/resource_aws_lb_target_group.go

```diff
@@ -158,6 +158,13 @@ func resourceAwsLbTargetGroup() *schema.Resource {
 							Optional:     true,
 							Default:      86400,
 							ValidateFunc: validation.IntBetween(0, 604800),
+							DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
Cool, didn't know about this.

Reply:
It may not be the best way to do this but I did not want to remove the default. At least it's easy to understand.